src/main/java/demohometask/demohometask/Team.java

```diff
@@ -0,0 +1,65 @@
+package demohometask.demohometask;
+
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+@Component
```

Comment:
This is stateful class that might have different instances in the application for different teams. It shouldn't be spring component